### PR TITLE
update readme for e2e testing.

### PR DESCRIPTION
**Pull request by morvencao**
Ref: https://github.com/open-cluster-management/backlog/issues/10140

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,29 +7,67 @@ This is a container which will be called from:
...
+
+The supported component images include:
+
+- multicluster-observability-operator
+- rbac-query-proxy
+- metrics-collector
+- endpoint-monitoring-operator
+- grafana-dashboard-loader
+
+> Note: the component image override is useful when you want to test each stockholder repositories, you only need to export the `COMPONENT_IMAGE_NAME` environment if running the e2e testing locally. For the CICD pipeline, the prow will take care of entirely process, thet means that when you raise a PR to the stockholder repositories, the prow will build the image based the source code of your commit and then install the Observability accordingly.
```
Comment (songleo) [inline README.md:52]: typo need to fixed (`thet`)

File: README.md [hunk 2]
```diff
@@ -7,29 +7,67 @@ This is a container which will be called from:
...
 The tests in this container will:
 
 1. Create the MCO CR . The Object store to be already in place for CR to work.
 2. Wait for the installation to complete.
 3. Then check the entire Observability suite (Hub and Addon) is working as expected including disable/enable, Grafana etc.
 
-## Running E2E
+## Setup E2E Testing Environment
+
+If you only have an OCP cluster and haven't installed Observability yet, then you can install the Observability(both Hub and Addon) by the following steps:
```
Comment (songleo) [inline README.md:18]: space is needed. (`Observability(both Hub and Addon)`)

**Review by songleo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (morvencao) [reply README.md]: thanks, done.

File: README.md (hunk 2 above)
Comment (morvencao) [reply README.md]: Done.

**Comment by morvencao** [comment]
@songleo Applied your comments, please review it again.

**Review by songleo — APPROVED** [review]

**Pull request merged**